### PR TITLE
[4.13] add new ci jobs coverage

### PR DESCRIPTION
**Pull request by jianlinliu**
Add new ci jobs coverage for 4.13 new features

### CODE REVIEW

**Comment by jianlinliu** [comment]
pj-rehearse

**Comment by jianlinliu** [comment]
cc @jianli-wei @jinyunma @liangxia @yunjiang29 to review

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -191,6 +191,29 @@ tests:
...
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
+- as: aws-ipi-compact-p2-f14
+  cron: 4 21 18,2 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.13'
+      TEST_ADDITIONAL: Network_Observability
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:202]: Is this intentionally ? Any context why we need to test `Network_Observability` on profile `aws-ipi-compact` ?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:202]: I just coped the lines from other existing jobs, I am going to remove it.

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: @jianlinliu AWS part is being added by #36163

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: oh, good to know, I am going to drop aws part in this PR.

**Pull request merged**